src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1250,6 +1250,11 @@ components:
           $ref: '#/components/schemas/AnonymousSupportingInformationFormat'
         deliveryAddressType:
           $ref: '#/components/schemas/DeliveryAddressType'
+        collectionReason:
+          type: string
+          description: |
+            The reason the elector gave for having to collect their voter ID (if applicable).
```

Comment:
Minor:  `.... to collect their AED (if applicable)`? Although AED is a voter ID

Reply:
Also worth updating api spec version

Reply:
Updated. I copied it from another collection reason this spec, which is for AEDs and VACs, but this one isn't 😄